psql2mysql/__init__.py

```diff
@@ -51,8 +51,9 @@ def lookupTypeDecorator(srcType, destType):
 
 
 class DbWrapper(object):
-    def __init__(self, uri=""):
+    def __init__(self, uri="", chunk_size=None):
```

Comment:
I think you should default to 0 here, not None.

Reply:
Defaulting to 0 would mean turning on non chunked migration by default. Well I guess it wouldn't so long as oslo config is around to give the fallback.